eclipse/che
# Refactor typescript DTO pom.xml

Signed-off-by: Mykhailo Kuznietsov <mkuznets@redhat.com>



### What does this PR do?
Couple of fixes to the typescriptd-to pom file
- adding snapshot repositories, to be able to build from snapshot versions of the plugin (just in case)
- setting a fixed che parent version for the project
- utilizing the `che.version` property, for simpler replacement of versions during the release


### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/17721


### How to test this PR?

run `mvn clean install -f dto-pom.xml` from `typescript-dto` directory. Prior to that, set any release or snapshot existing version of Che for `che.version` property of the dto-pom.xml. Build should succeed.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
